core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -450,6 +458,33 @@ trait EntityComponent {
         val entityTypeNameTuples = reduceSqlActionsWithDelim(entities.map { ref => sql"(${ref.entityType}, ${ref.entityName})" })
         concatSqlActions(baseUpdate, entityTypeNameTuples, sql")").as[Int]
       }
+
+      def batchHideForType(workspaceContext: Workspace, entityType: String): ReadWriteAction[Seq[Int]] = {
```

Comment:
nit: `batchHideAttributesOfType`? `batchHideAttributes`? As-is, `batchHideForType` and `batchHideType` feel too confusingly similar

Reply:
Renamed in 043714445336abf20e2b4c591ae74182077e2a2f